src/explore-education-statistics-admin/src/pages/publication/PublicationManageTeamAccessPage.tsx

```diff
@@ -35,10 +35,11 @@ const PublicationManageTeamAccessPage = ({
   const [currentReleaseId, setCurrentReleaseId] = useState(releaseId ?? '');
 
   const { value: model, isLoading } = useAsyncHandledRetry<Model>(async () => {
-    const [releases, publication] = await Promise.all([
+    const [paginatedReleases, publication] = await Promise.all([
```

Comment:
I'd maybe not call these paginatedReleases when we're getting all of them just because it's a bit confusing, I was reading this and thinking that it'd just get a page of them (or possibly I've not had enough coffee yet!).

Reply:
Discussed this with Amy on Slack. Could go for `allReleases` or `allReleasesSinglePage`. Not sure if others have an opinion?

Reply:
I'd probably vote for `allReleases` in this instance

Reply:
Done